### PR TITLE
Initiation

### PR DESCRIPTION
**Pull request by RNAer**

### CODE REVIEW

File: micronota/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, micronota development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+__credits__ = "https://github.com/biocore/micronota/graphs/contributors"
+__version__ = "0.1.0.dev0"
```
Comment (antgonza) [inline micronota/__init__.py]: Should this be `__version__ = "0.1.0-dev"`?

File: micronota/cli.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+from os import listdir
+from os.path import abspath, join, dirname, splitext
+
+
+_CONTEXT_SETTINGS = dict(
+    # allow insensitivity to letter task of the command options
```
Comment (antgonza) [inline micronota/cli.py]: allow insensitivity to letter task of the -> allow case insensitive?

**Comment by antgonza** [comment]
Couple of comments.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      license='BSD',
+      description=description,
+      long_description=long_description,
+      keywords=keywords,
+      classifiers=classifiers,
+      author="micronota development team",
+      author_email="zhenjiang.xu@gmail.com",
+      maintainer="micronota development team",
+      maintainer_email="zhenjiang.xu@gmail.com",
+      url='http://micronota.org',
```
Comment (antgonza) [inline setup.py]: should this be http://microbio.me/micronota?

File: micronota/__init__.py (hunk 1 above)
Comment (RNAer) [inline micronota/__init__.py]: `dev0` is actually one of standard way to version string: https://www.python.org/dev/peps/pep-0440/

File: setup.py (hunk 3 above)
Comment (RNAer) [inline setup.py]: that may be better.

**Comment by RNAer** [comment]
Thanks, @antgonza ! If there is no problem with the update, merge it? I guess the policy should be that the author of a pull request is not supposed to do the merge.

File: micronota/cli.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+_CONTEXT_SETTINGS = dict(
+    # allow insensitivity to letter case of the command options
+    token_normalize_func=lambda x: x.lower(),
+    # set --help option for all (sub)commands
+    help_option_names=['-h', '--help'])
+
+
+class AliasedGroup(click.Group):
+    ''' Custom subclass of click.Group to enable alias for commands.
```
Comment (antgonza) [inline micronota/cli.py]: Extra space here.

File: micronota/cli.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import click
+from os import listdir
+from os.path import abspath, join, dirname, splitext
+
+
+_CONTEXT_SETTINGS = dict(
+    # allow insensitivity to letter case of the command options
```
Comment (antgonza) [inline micronota/cli.py]: This still needs some fixing.

File: micronota/cli.py (hunk 4 above)
Comment (RNAer) [inline micronota/cli.py]: extra space where?

Comment (antgonza) [inline micronota/cli.py]: '''[space] Custom subclass 

Sorry for not being clear.

Comment (RNAer) [inline micronota/cli.py]: ah, it's so hard to notice. thanks.

**Comment by mortonjt** [comment]
I don't see any outstanding problems with this PR.  What needs to be done to merge this?

**Comment by RNAer** [comment]
just merge it?

**Comment by antgonza** [comment]
Note that we don't get a message when you do a new commit, except if you mention who should get it, for example @antgonza. Merging now.

**Comment by RNAer** [comment]
:+1:

**Pull request merged**